topics/web-components/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: webcomponent, web-component, webcomponents
+created_by: Alex Russell
+display_name: Web Components
+github_url: https://github.com/webcomponents
+logo: web-components.png
+related: custom-elements, shadow-dom, es-modules, html-templates, html-imports
+short_description: Web Components are a set of web platform APIs developers can use to create custom HTML tags.
+topic: web-components
+url: https://www.webcomponents.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Web_Components
```

Comment:
Based on the Wikipedia page, you could add a `released: 2011` line if you wanted.